packages/formstr-app/src/containers/Dashboard/index.tsx

```diff
@@ -176,11 +169,6 @@ export const Dashboard = () => {
           </Dropdown>
         </div>
         <div className="form-cards-container">{renderForms()}</div>
-        <TemplateSelectorModal
```

Comment:
now we don't have templates on empty screen? I loved that, why did we remove it?